xradar/model.py

```diff
@@ -286,6 +296,12 @@
         ("system_phidp", None),
         ("test_power_h", None),
         ("test_power_v", None),
+        ("tx_pulse_blind_length", None),  # Furuno
+        ("tx_pulse_specification", None),  # Furuno
+        ("tx_power_h", None),
+        ("tx_power_v", None),
```

Comment:
Would this make sense?

```suggestion
        ("tx_power_h", "xmit_power_h"),
        ("tx_power_v", "xmit_power_v"),
```